packages/Python/lldbsuite/test/lang/swift/implementation_only_imports/main_executable_resilient/TestMainExecutableResilient.py

```diff
@@ -0,0 +1,86 @@
+# TestMainExecutableResilient.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2019 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
+"""
+Test `@_implementationOnly import` in the main executable with a resilient library
+"""
+import commands
+import lldb
+from lldbsuite.test.lldbtest import *
+import lldbsuite.test.decorators as decorators
+import lldbsuite.test.lldbutil as lldbutil
+import os
+import os.path
+import time
+import unittest2
+
+class TestMainExecutable(TestBase):
+
+    mydir = TestBase.compute_mydir(__file__)
+
+    def launch_and_break(self):
+        exe_name = "main"
+        exe_path = self.getBuildArtifact(exe_name)
+        target = self.dbg.CreateTarget(exe_path)
+        self.assertTrue(target, VALID_TARGET)
+        
+        source_name = "main.swift"
+        source_spec = lldb.SBFileSpec(source_name)
+        breakpoint = target.BreakpointCreateBySourceRegex("break here", source_spec)
+        self.assertTrue(breakpoint.GetNumLocations() > 0, "Breakpoint set sucessfully")
+        
+        process = target.LaunchSimple(None, None, os.getcwd())
+        self.assertTrue(process, PROCESS_IS_VALID)
+    
+    @decorators.swiftTest
+    def test_implementation_only_import_main_executable_resilient(self):
+        """Test `@_implementationOnly import` in the main executable with a resilient library"""
```

Comment:
Could you please just add a sentence or two outlining why `@_implementationOnly` is special and how it is expected to deviate or not form the normal behavior?
Inevitable at some point somebody else will have to do maintenance on the test and without a comment explaining what behavior needs to be preserved, it can easily happen that the test devolves into something that isn't meaningful any more.